### PR TITLE
Change header parser

### PR DESCRIPTION
**Pull request by ovainola**
Changed the header parser to check for comment lines and
to search for given regular expressions. Also changed println
function into info functions

### CODE REVIEW

**Comment by ahojukka5** [comment]
Could you rebase? I  set up Travis-CI and added one more test.

**Comment by ovainola** [comment]
Sure, I'll rebase after you've made a new commit

**Comment by ahojukka5** [comment]
Heh, I forgot. Now is.

File: src/CheckHeader.jl [hunk 1]
```diff
@@ -1,29 +1,62 @@
-# This file is a part of JuliaFEM.
+# This file is a part of JuliaFEM. 
 # License is MIT: see https://github.com/ahojukka5/CheckHeader.jl/blob/master/LICENSE
 
 module CheckHeader
 
-function checkheader(package::String)
-    end_idx = 2
+"""Read header from file content
```
Comment (ahojukka5) [inline src/CheckHeader.jl]: To get docstring well formatted, the first line should be indented by 4 spaces, after that oneliner what function does + rest. See [here](https://docs.julialang.org/en/stable/manual/documentation/).

File: src/CheckHeader.jl [hunk 2]
```diff
@@ -1,29 +1,62 @@
...
+
+Searches "part of" and "license" lines using regular expression
+from file header
+"""
+function read_header(fd)
+    lines = readlines(fd)
+    header_comment = true
+    header_lines = []
+    regex_strings = [r"(This file is a part of \w+)",
```
Comment (ahojukka5) [inline src/CheckHeader.jl]: Actually, is this now more general than before? Now we expect that lines must contain some specific words. All we know is that they must start with risuaita.

Comment (ovainola) [inline src/CheckHeader.jl]: It searches for the start and license strings but is not restricted to two lines. From what I checked many julia packages headers may vary quite a bit, so should be change that first find all comment lines, from those extract "part of" and "license" and least verify that those lines are found.

File: src/CheckHeader.jl (hunk 1 above)
Comment (ovainola) [inline src/CheckHeader.jl]: Yes, this is not according to the style described in link, I'll fix it

**Pull request closed without merging**